Docs/Configuration.tex

```diff
@@ -2766,126 +2775,135 @@ \section{Misc}\label{misc}
-section for more details.
+The scan process starts with enumerating all available partitions, filtered based on the \texttt{Scan policy}.
+Each partition may generate multiple primary and alternate options. Primary options represent operating systems
+installed on the media while alternate options represent recovery options for the operating systems on the media.
+\begin{itemize}
+\tightlist
+\item Alternate options may exist without primary options and vice versa.
+\item Options may not necessarily represent operating systems on the same partition.
+\item Each primary and alternate option can be an auxiliary option or not.
+\begin{itemize}
+\tightlist
+  \item Refer to the \texttt{HideAuxiliary} section for more details.
+\end{itemize} \medskip
+\end{itemize}
 
 The algorithm to determine boot options behaves as follows:
 
 \begin{enumerate}
 \tightlist
-\item Obtain all available partition handles filtered by ``Scan policy'' (and driver availability).
-\item Obtain all available boot options from \texttt{BootOrder} UEFI variable.
-\item For each found boot option:
+\item Obtain all available partition handles filtered based on the \texttt{Scan policy} (and driver availability).
+\item Obtain all available boot options from the \texttt{BootOrder} UEFI variable.
+\item For each boot option found:
   \begin{itemize}
-  \item Retrieve device path of the boot option.
+  \item Retrieve the device path of the boot option.
   % Scan policy restrictions are actually checked here as we want the function to be self-contained
   % for non-scan based startup.
   \item Perform fixups (e.g. NVMe subtype correction) and expansion (e.g. for Boot Camp) of the device path.
-  \item Obtain device handle by locating device path of the resulting device path (ignore it on failure).
-  \item Find device handle in the list of partition handles (ignore it if missing).
-  % To determine device path type we can use LocateDevicePath RemainingDevicePath argument. Just check whether
-  % it points to the END device path.
-  \item For disk device paths (not specifying a bootloader) execute ``bless'' (may return > 1 entry).
-  \item For file device paths check presence on the file system directly.
+  \item Obtain the device handle by locating the device path of the resulting device path (ignore it on failure).
+  \item Locate the device handle in the list of partition handles (ignore it if missing).
+  % To determine device path type we can use LocateDevicePath RemainingDevicePath argument.
+  % Just check whether it points to the END device path.
+  \item For disk device paths (not specifying a bootloader), execute ``bless'' (may return > 1 entry).
+  \item For file device paths, check for presence on the file system directly.
   % Just kill all \EFI\APPLE\ paths.
-  \item On OpenCore boot partition exclude all OpenCore bootstrap files by header checks.
+  \item On the OpenCore boot partition, exclude all OpenCore bootstrap files by header checks.
```

Comment:
Sorry, I am going to have my breakfast now. Let me leave a mark here so that I would be back to it once finished. :P